taxi/tests/test_models.py

```diff
@@ -0,0 +1,55 @@
+from django.contrib.auth import get_user_model
+from django.test import TestCase
+from django.urls import reverse
+
+from taxi.models import Manufacturer, Driver, Car
+
+
+class ModelTests(TestCase):
+    def test_manufacturer_str(self):
+        manufacturer = Manufacturer.objects.create(name="test",
+                                                   country="test")
+        self.assertEqual(str(manufacturer),
+                         f"{manufacturer.name} {manufacturer.country}")
+
+    def test_driver_str(self):
+        driver = get_user_model().objects.create(username="test",
+                                                 first_name="test",
+                                                 last_name="test")
+        self.assertEqual(str(driver),
+                         f"{driver.username} ({driver.first_name} "
+                         f"{driver.last_name})")
+
+    def test_car_str(self):
+        car = Car.objects.create(model="test",
+                                 manufacturer=Manufacturer.objects.create(
+                                     name="test", ))
```

Comment:
The `Manufacturer` object is created without a `country` field. If the `country` field is required by the `Manufacturer` model, you should provide a value for it to avoid potential errors.